DH_Engine/Classes/DHGameReplicationInfo.uc

```diff
@@ -792,6 +793,35 @@ simulated function string GetTeamScaleString(int Team)
    }
 }
 
+// Modified to allow VoiceID to be greater than 32
+simulated function AddPRI(PlayerReplicationInfo PRI)
+{
+    local byte      NewVoiceID;
+    local int       i;
+
+    if (Level.NetMode == NM_ListenServer || Level.NetMode == NM_DedicatedServer)
+    {
+        for (i = 0; i < PRIArray.Length; i++)
+        {
+            if (PRIArray[i].VoiceID == NewVoiceID)
+            {
+                i = -1;
```

Comment:
Not sure why this is needed. Upon next iteration, this loop body will just do nothing since it'll be evaluating the same record against a different value.